### PR TITLE
Clone branch

### PR DESCRIPTION
**Pull request by bren2102**
I Kindly request a code review for my [Newsweek](https://www.newsweek.com/) clone project which I developed by myself using HTML CSS and the bootstap framework.

- Added and styled Top Section
- Added and styled Main Section
- Added and styled Footer Section
- Added and styled Navbar
- Added and styled Images
- Added and styled Sticky Sections

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1776 @@
...
+      </li>
+      <li class="nav-item border-right">
+        <a class="nav-link text-reset" href="#">Vantage</a>
+      </li>
+      <li class="nav-item border-right-0">
+        <a class="nav-link text-reset" href="#">Weather</a>
+      </li>
+      <li class="nav-item border-right-0">
+        <form class="form-inline">
+          <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search"  style="width: 90px; font-size: 12px;">
```
Comment (pelzolga123) [inline index.html]: No inline styling. Please move all styles from `index.html` to `style.css`

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+
+div,
+p {
+  line-height: 15px;
+}
+
+/* Header starts */
+header {
+  /* width: 1300px;
+  margin: 0 auto; */
```
Comment (pelzolga123) [inline CSS/style.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Comment by pelzolga123** [comment]
## Hi Brenda!  
### Your project looks :+1: great!
Here is a list of things to change, before this project can be approved:
* [ ] Navigation bar needs to look more like original one:
* Laptop size:
image

Original:
image

* Mobile size:
image

Original:
image

* [ ] Some of your sections have very small distance between them, that needs to be corrected to look more like original. For example:
image

Original:
image

* [ ] Next section should have same height and `newsletter` part should look more like original in large, tablet and mobile versions:
image

Original:
image

* [ ] Next two sections require adjustments in mobile version:
image

Original:
image

* [ ] Footer in mobile version should look more like original:
image

Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by bren2102** [comment]
- No styling on HTML
- Comments deleted on CSS
- Navigation bar on mobile and web fixed 
- Distance between sections fixed
- Newsweek and Newsletter section fixed
- Latest News an Opinion section fixed
- Footer Fixed

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)

**Comment by nikasulo** [comment]
# Fantastic work @bren2102 :clap: 
You have done a great job building this and incorporating the changes suggested by @pelzolga123 :medal_military: Please consider my comments below, make necessary changes and request a re-review :100: 

#### Responsiveness
Your project could use a little more responsiveness, please look a little more closely at the sizes of elements, padding of sections, margins, and general element positionings. These are very key as they will affect the look, feel, and more importantly the usability of your sites. I went ahead to attach a few images of areas where I noticed issues with responsiveness, however, please do a full scan of your page just in case I missed or left something out.

Please note the following responsive issues:

#### Navbar && Opinions sections
Please take note of how the nav links aren't sitting properly and the opinions section also has a few bugs
screenshot

#### Categories
Please notice the tech and science tag not sitting properly. Please calculate for the length of longer texts in your layout or font sizing
screenshot

These errors were found at 1024px

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by bren2102** [comment]
- Navbar fixed
- Opinion space section fixed
- Categories text section

**Review by rOluochKe — APPROVED** [review]
Hello Team!
### Good job, I see you have made changes from previous reviews and your project looks good.
### Congratulations your project has been approved :clap: :boom:

- You can merge and submit a project completion form from your dashboard
- All the best in your next project.
- Happy coding!

**Pull request merged**